README.md

```diff
@@ -1,8 +1,6 @@
-#################
-    COPYRIGHT
-#################
+## Copyright
```

Comment:
Level-2 headings can also be created in Markdown by underlining the text with a row of hyphens. (A row of = signs will produce H1 headings.) I wonder if that might not make them "stand out" more in the _source_ text file? The text can even be indented (up to 3 spaces). These will all produce the same H2-level "Copyright" heading:
```md
## Copyright

Copyright
---------
 
   Copyright
---------------
```

  Copyright
-------------